test/setup/DeviceServerMock.js

```diff
@@ -0,0 +1,11 @@
+// @flow
```

Comment:
keep it very simple for now, until I figure out how to use mock libs and what them for.